source/deployment/webrtc.rst

```diff
@@ -0,0 +1,57 @@
+Video and Audio Calling (Beta)
+=====================================
+
+Mattermost supports an early preview of video and audio calling option using a self-hosted proxy. The implementation is based on `WebRTC integration <https://www.html5rocks.com/en/tutorials/webrtc/basics/>`_ enabling one-on-one video and audio chat without plugins.
+
+The feature is currently intended as a working prototype for community development and not recommended for production. It is supported on Chrome, Firefox and the Mattermost Desktop Apps.
+
+Configuring video and audio calls
+--------------------------
+
+This option can be enabled by the System Administrator in the System Console under **Integrations > WebRTC (Beta)** - `see configuration settings documentation to learn more <https://docs.mattermost.com/administration/config-settings.html#webrtc-beta>`_.
+
+To set up the WebRTC server, you may either
+
+ - use a `Mattermost docker container created for testing WebRTC <https://hub.docker.com/r/mattermost/webrtc/>`_
+ - set up a `Janus <https://github.com/meetecho/janus-gateway>`_ or `Coturn <https://github.com/coturn/coturn/wiki>`_ gateway server for your Mattermost installation.
+
+Starting a video call
+--------------------------
+
+After enabling the feature:
```

Comment:
Maybe clarify that this is after enabling in the system console done by the system admin